### ISSUE TITLE
Ensure that the colors used on the IT standards report filters are Section 508 compliant

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#798]
Josh stated that the colors within the table itself should be Section 508 compliant, but are the colors used in the report filters of the IT standards report also Section 508 compliant?
